tests/automated_benchmarking.py

```diff
@@ -0,0 +1,220 @@
+import argparse
+import glob
+import json
+import os
+import pickle as pk
+import subprocess
+import urllib.request
+
+
+GITHUB_API_PR_URL = "https://api.github.com/repos/facebook/zstd/pulls?state=open"
+GIRHUB_URL_TEMPLATE = "https://github.com/{}/zstd"
```

Comment:
`GIRHUB` or `GITHUB` ?

Reply:
Ah its supposed to be used in line 68 but it looks like I forgot. And yes, GITHUB:)